### PR TITLE
Settings and Permissions

### PR DESCRIPTION
**Pull request by pveland621**

### CODE REVIEW

File: src/sass/_profile.scss [hunk 1]
```diff
@@ -1078,6 +1078,5 @@ $padding-no-border: $total-padding - $border;
 
   .header {
     font-weight: bold;
-    text-align: center;
```
Comment (elizabeth-jimenez) [inline src/sass/_profile.scss]: 😂 🧑‍🎨

File: src/Components/Settings/Settings.jsx [hunk 2]
```diff
@@ -1,57 +1,116 @@
...
+        <CopyToClipboard
+          text={getTokens()}
+          onCopy={() => setCopied(true)}
+        >
+          <button>Copy to clipboard</button>
+        </CopyToClipboard>
+        {copied ? <span>Copied!</span> : null}
+      </div>
+      <div>
+        <table className="permissions-table">
```
Comment (elizabeth-jimenez) [inline src/Components/Settings/Settings.jsx:53]: Let's merge these 3 tables into 1 - we should be able to keep it looking the same by playing with the `margin` in `.header`

**Comment by elizabeth-jimenez** [comment]
You'll want to update the test file `Settings.test.jsx`. One of the steps is to wrap it in Provider. Check out `EmployeeAgendaSearch.test.jsx` as a template. Another step is that you'll have to pass it fake props - checkout the fake props in `FeatureFlags.test.jsx` as a template

**Comment by pveland621** [comment]
> You'll want to update the test file `Settings.test.jsx`. One of the steps is to wrap it in Provider. Check out `EmployeeAgendaSearch.test.jsx` as a template. Another step is that you'll have to pass it fake props - checkout the fake props in `FeatureFlags.test.jsx` as a template

I saw that Settings.jsx doesn't use any props, so I worked with Cory and we agreed that you probably meant to pass in fake data for the roles, bureau permissions, and org permissions arrays, which I implemented. Otherwise, the test file is complete and passing.

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!

**Pull request merged**